qutip/tests/core/data/test_reshape.py

```diff
@@ -0,0 +1,135 @@
+from .test_mathematics import UnaryOpMixin
+import pytest
+import numpy as np
+from qutip import data
+from qutip.core.data import CSR, Dense
+
+
+class TestSplitColumns(UnaryOpMixin):
+    def op_numpy(self, matrix):
+        return [matrix[:, i].reshape((-1, 1)) for i in range(matrix.shape[1])]
+
+    specialisations = [
+        pytest.param(data.split_columns_csr, CSR, list),
+        pytest.param(data.split_columns_dense, Dense, list),
+    ]
+
+
+class TestColumnStack(UnaryOpMixin):
+    # It seem column_stack assumes F ordered array?
+    def op_numpy(self, matrix):
+        out_shape = (matrix.shape[0]*matrix.shape[1], 1)
+        return np.reshape(matrix, newshape=out_shape, order='F')
+
+    specialisations = [
+        pytest.param(data.column_stack_csr, CSR, CSR),
+        pytest.param(data.column_stack_dense, Dense, Dense),
+    ]
+
+
+class TestColumnUnstack(UnaryOpMixin):
+    # It seem column_unstack assumes F ordered array?
+    def op_numpy(self, matrix, rows):
+        out_shape = (rows, matrix.shape[0]*matrix.shape[1]//rows)
+        return np.reshape(matrix, newshape=out_shape, order='F')
+
+    shapes = [
+        (pytest.param((10, 1), id="ket"), ),
+    ]
+
+    bad_shapes = [
+        (pytest.param((1, 10), id="bra"), ),
+        (pytest.param((10, 10), id="square"), ),
+        (pytest.param((2, 10), id="non_square"), ),
+    ]
+
+    specialisations = [
+        pytest.param(data.column_unstack_csr, CSR, CSR),
+        pytest.param(data.column_unstack_dense, Dense, Dense),
+    ]
+
+    # `ColumnUnstack` has an additional scalar parameters. This is why tests
+    # are a little bit more involved.
```

Comment:
This comment made sense when it was just one comment on one test (`add`, maybe?) in the original file.  Now that it's pretty common, this is just noise and should be removed.